guides/common/modules/con_network-boot-provisioning-workflow.adoc

```diff
@@ -80,11 +80,3 @@ If you use image-based provisioning, the host does not follow the standard PXE b
 The compute resource creates a copy of the image for the host to use.
 Depending on image settings in {Project}, seed data can be passed in for initial configuration, for example using `cloud-init`.
 {Project} can connect using SSH to the host and execute a template to finish the customization.
-+
-[NOTE]
-====
-By default, deleting the provisioned profile host from {Project} does not destroy the actual VM on the external compute resource.
-To destroy the VM when deleting the host entry on {Project}, navigate to *Administer* > *Settings* > *Provisioning* and configure this behavior using the *destroy_vm_on_host_delete* setting.
-If you do not destroy the associated VM and attempt to create a new VM with the same resource name later, it will fail because that VM name already exists in the external compute resource.
-You can still register the existing VM to {Project} using the standard host registration workflow you would use for any already provisioned host.
-====
```

Comment:
Consider linking the new module in a note here.

Reply:
This module is inappropriate even for the link. I've added it to chapters with VMs below instead.